build.sbt

```diff
@@ -64,10 +64,10 @@ lazy val playJsonTests = project in file("playJsonTests") settings(common: _*) s
   name := "play-json-tests",
 
   libraryDependencies ++= Seq(
-    "com.typesafe.play" %% "play-json" % "2.3.7",
-    "org.scalacheck" %% "scalacheck" % "1.12.2",
-    "org.scalatest" %% "scalatest" % "2.2.4",
-    "me.jeffmay" %% "scalacheck-ops" % "1.0.0"
+    "com.typesafe.play" %% "play-json" % "2.3.9",
+    "org.scalacheck" %% "scalacheck" % "1.12.5",
+    "org.scalatest" %% "scalatest" % "2.2.6",
```

Comment:
It's still a milestone and not a release. Also, these updates are going to be the latest on the 2.10 branch. After these changes I plan to remove support for 2.10 and upgrade everything to the latest major versions.